docs/source/pythonapi/index.rst

```diff
@@ -185,6 +185,7 @@ Running OpenMC
    :template: myfunction.rst
 
    openmc.run
+   openmc.run_volume_calculation
```

Comment:
I'm ok with that. I chose "run" as some kind of indication that the Fortran code is being run, but "plot_geometry" already breaks that convention.